airflow/providers/apache/cassandra/example_dags/example_cassandra_dag.py

```diff
@@ -35,7 +35,9 @@
     catchup=False,
     tags=['example'],
 ) as dag:
-    table_sensor = CassandraTableSensor(task_id="cassandra_table_sensor")
+    table_sensor = CassandraTableSensor(task_id="cassandra_table_sensor", table="table")
 
-    record_sensor = CassandraRecordSensor(task_id="cassandra_record_sensor", keys={"p1": "v1", "p2": "v2"})
+    record_sensor = CassandraRecordSensor(
+        task_id="cassandra_record_sensor", keys={"p1": "v1", "p2": "v2"}, table="table"
+    )
```

Comment:
I think we should keep the `table` arg value in `default_args` and add the `type: ignore[call-arg]` comment instead. It seems like a proper fix is not going to be trivial but ignoring that specific Mypy error is a good interim solution.

The documentation for these sensors also references that the `table` parameter is set in `default_args`.

cc @uranusjr @khalidmammadov (related your Slack conversation)

Reply:
Maybe then we should make it simply optional and raise Exception if it is not set. That seems like the best approach and very simple to do.

Reply:
@josh-fell  - I pushed a fix with that. I thin it's simple and straightforward with Optional table and checking at runtime.

Reply:
Random idea I just played around with that seemed for work here. What if we created a stub file for the example DAG?

`airflow/providers/apache/cassandra/example_dags/example_cassandra_dag.pyi`
```python
from typing import Any, Dict

from airflow.providers.apache.cassandra.hooks.cassandra import CassandraHook
from airflow.providers.apache.cassandra.sensors.record import CassandraRecordSensor as _CassandraRecordSensor
from airflow.providers.apache.cassandra.sensors.table import CassandraTableSensor as _CassandraTableSensor


class CassandraTableSensor(_CassandraTableSensor):
    def __init__(
        self, *, table: str = ..., cassandra_conn_id: str = CassandraHook.default_conn_name, **kwargs: Any
    ) -> None:
        ...

class CassandraRecordSensor(_CassandraRecordSensor):
    def __init__(
        self,
        *,
        table: str = ...,
        keys: Dict[str, str] = ...,
        cassandra_conn_id: str = CassandraHook.default_conn_name,
        **kwargs: Any,
    ) -> None:
        ...

```
We don't have to change the operator logic or the example DAG itself and Mypy seems happy.

Reply:
FYI - Just simplified the above stub file a little.